src/imodmodel/models.py

```diff
@@ -264,4 +317,50 @@ def from_file(cls, filename: os.PathLike):
         from .parsers import parse_model
         with open(filename, 'rb') as file:
             return parse_model(file)
+    
+    def to_file(self, filename: os.PathLike):
+        """Write an IMOD model to disk."""
+        from .writers import write_model
+        with open(filename, 'wb') as file:
+            write_model(file, self)
+
+    @classmethod
+    def new(cls):
```

Comment:
This looks a bit like C++? 

What's the purpose of this classmethod? Shouldn't this be doable using defaults and the default Pydantic `__init__`, especially as it seems to be a general purpose constructor method?

Reply:
Hey, some of us have to put up with C++ and it messes with your head...

Your absolutely right, though

Reply:
I think it's justifiable to have multiple different default constructors as classmethods, but I would probably name them for their intended purpose. Something like `new_scattered_model()`.